### PR TITLE
#4727 fix up the view server postman collection

### PR DESCRIPTION
**Pull request by davidradl**

### CODE REVIEW

**Review by wbittles — APPROVED** [review]

File: open-metadata-implementation/admin-services/Egeria-admin-services-server-configuration.postman_collection.json [hunk 1]
```diff
@@ -1952,7 +1952,7 @@
 								],
 								"body": {
 									"mode": "raw",
-									"raw": "      { \n         \"class\":\"ViewServiceConfig\",\n         \"omagserverName\":\"cocoMDS1\",\n         \"omagserverPlatformRootURL\":\"https://localhost:9443\"\n      }"
+									"raw": "      { \n         \"class\":\"ViewServiceConfig\",\n         \"omagserverName\":\"myMetadataAccessPoint\",\n         \"omagserverPlatformRootURL\":\"myPlatformRootURL1\"\n      }"
```
Comment (planetf1) [inline open-metadata-implementation/admin-services/Egeria-admin-services-server-configuration.postman_collection.json:1955]: This should be using postman variables from your environment definition ie  {{platformURL}} {{serverName}} etc
Use the variables defined in the current env files & extend with any new concepts like view servers etc - that way someone just needs to update their environment definition, not edit the actual collection?

File: open-metadata-implementation/admin-services/Egeria-admin-services-server-configuration.postman_collection.json [hunk 2]
```diff
@@ -1987,7 +1987,7 @@
...
...on\"\n        },       \n        {\n            \"class\"              : \"ResourceEndpointConfig\",\n            \"resourceCategory\"   : \"Server\",\n            \"serverInstanceName\" : \"myServerInstanceName1\",\n            \"serverName\"         : \"myServerName1\",\n            \"platformName\"       : \"myOMAGPlatform1\",\n            \"description\"        : \"myServerName1 description \"\n        },\n        {\n            \"class\"              : \"ResourceEndpointConfig\",\n            \"resourceCategory\"   : \"Server\",\n            \"serverInstanceName\" : \"myServerInstanceName2\",\n            \"serverName\"         : \"myServerName2\",\n            \"platformName\"       : \"myOMAGPlatform1\",\n            \"description\"        : \"myServerName2 description \"\n        },\n        {\n            \"class\"              : \"ResourceEndpointConfig\",\n            \"resourceCategory\"   : \"Server\",\n            \"serverInstanceName\" : \"myServerInstanceName3\",\n            \"serverName\"         : \"myServerName3\",\n            \"platformName\"       : \"myOMAGPlatform2\",\n            \"description\"        : \"myServerName3 description \"\n        }\n    ]\n}"
```
Comment (planetf1) [inline open-metadata-implementation/admin-services/Egeria-admin-services-server-configuration.postman_collection.json:1990]: should also use variables

File: open-metadata-implementation/admin-services/Egeria-admin-services-server-configuration.postman_collection.json [hunk 3]
```diff
@@ -2022,7 +2022,7 @@
...
```
Comment (planetf1) [inline open-metadata-implementation/admin-services/Egeria-admin-services-server-configuration.postman_collection.json:2025]: variables...

File: open-metadata-implementation/admin-services/Egeria-admin-services-server-configuration.postman_collection.json [hunk 4]
```diff
@@ -2057,7 +2057,7 @@
...
+									"raw": "{\n    \"class\":\"IntegrationViewServiceConfig\",\n    \"viewServiceAdminClass\":\"org.odpi.openmetadata.viewservices.serverauthor.admin.ServerAuthorViewAdmin\",\n    \"viewFullServiceName\":\"ServerAuthor\",\n    \"viewServiceOperationalStatus\":\"ENABLED\",\n    \"omagserverPlatformRootURL\":\"myPlatformRootURL1\",\n    \"resourceEndpoints\" : [\n        {\n            \"class\"              : \"ResourceEndpointConfig\",\n            \"resourceCategory\"   : \"Platform\",\n            \"platformName\"       : \"myOMAGPlatform1\",\n            \"platformRootURL\"    : \"myPlatformRootURL1\",\n            \"description\"        : \"myOMAGPlatform1 description\"\n        },\n        {\n            \"class\"              : \"ResourceEndpointConfig\",\n            \"resourceCategory\"   : \"Platform\",\n            \"platformName\"       : \"myOMAGPlatform2\",\n            \"platformRootURL\"    : \"myPlatformRootURL2\",\n            \"description\"        : \"myOMAGPlatform2 description\"\n        }\n    ]\n}"
```
Comment (planetf1) [inline open-metadata-implementation/admin-services/Egeria-admin-services-server-configuration.postman_collection.json:2060]: variables

File: open-metadata-implementation/admin-services/Egeria-admin-services-server-configuration.postman_collection.json [hunk 5]
```diff
@@ -2092,7 +2092,7 @@
...
...on\"\n        },       \n        {\n            \"class\"              : \"ResourceEndpointConfig\",\n            \"resourceCategory\"   : \"Server\",\n            \"serverInstanceName\" : \"myServerInstanceName1\",\n            \"serverName\"         : \"myServerName1\",\n            \"platformName\"       : \"myOMAGPlatform1\",\n            \"description\"        : \"myServerName1 description \"\n        },\n        {\n            \"class\"              : \"ResourceEndpointConfig\",\n            \"resourceCategory\"   : \"Server\",\n            \"serverInstanceName\" : \"myServerInstanceName2\",\n            \"serverName\"         : \"myServerName2\",\n            \"platformName\"       : \"myOMAGPlatform1\",\n            \"description\"        : \"myServerName2 description \"\n        },\n        {\n            \"class\"              : \"ResourceEndpointConfig\",\n            \"resourceCategory\"   : \"Server\",\n            \"serverInstanceName\" : \"myServerInstanceName3\",\n            \"serverName\"         : \"myServerName3\",\n            \"platformName\"       : \"myOMAGPlatform2\",\n            \"description\"        : \"myServerName3 description \"\n        }\n    ]\n}"
```
Comment (planetf1) [inline open-metadata-implementation/admin-services/Egeria-admin-services-server-configuration.postman_collection.json:2095]: variables

**Review by planetf1 — CHANGES_REQUESTED** [review]
I think this collection needs to use variables from the environment, just changing the name is 
a marginal improvement, but we should be consistent with the other collections?

Great to have a collection for the view server though :-)

**Comment by planetf1** [comment]
The PR now seems to be checking correctly - backlog now mostly addressed

**Pull request merged**